.gdn/.gitignore

```diff
@@ -0,0 +1,10 @@
+﻿## Ignore Guardian internal files
+.r/
+rc/
+rs/
+i/
+p/
+c/
+
+## Ignore Guardian Local settings
+LocalSettings.gdn.json
```

Comment:
where possible, please end files in this repo with a \n

Reply:
everything under the .gdn file was automatically generated.  We can, however, add new lines at the end

Reply:
Removed all subfolders, retested, and it worked.  Committed changed to this PR.

Reply:
Added \n to end